uxarray/grid/grid.py

```diff
@@ -1340,6 +1342,27 @@ def node_face_connectivity(self, value):
         assert isinstance(value, xr.DataArray)
         self._ds["node_face_connectivity"] = value
 
+    @property
+    def face_edge_nodes_xyz(self):
```

Comment:
Not sure about the motive, but adding this to grid properties would mean additional memory usage of `3 * 2 * max_edges * num_faces`, which seems huge for high-res grids. It'd be a better memory utilization to use the `face_edge_connectivity`, which should be six times smaller in size, to compute such data, i.e. face_edge_nodes, on the run and then probably get it discarded (likely with the function's scope ends).

Reply:
Yeah that sounds like a better implementation. For the 3.75km grid, it would over 10GB to store this array, which is not practical at all.